base/libgit2/tag.jl

```diff
@@ -41,3 +41,5 @@ function target(tag::GitTag)
     oid_ptr == C_NULL && throw(Error.GitError(Error.ERROR))
     return GitHash(oid_ptr)
 end
+
+Base.show(io::IO, tag::GitTag) = print(io, "GitTag:\nTag name: $(name(tag)) target: $(target(tag))")
```

Comment:
kinda inconsistent to have one of these use interpolation and the other use multi-arg print